lib/testing_utilities/helpers.js

```diff
@@ -0,0 +1,19 @@
+import renderer from 'react-test-renderer'
+
+export const setupWithLayout = (component, layout) => {
```

Comment:
this should probably be more descriptive, and ideally be typed - I wouldn't be able to know from the name what objects to put in, perhaps something like?

```js
/** Renders a React Component after applying a nativeEvent to an onLayout function passed in to props */
export const jsonAfterApplyingNativeLayoutDimentions = (component: any, layout: { width?: number, height?:number } ) => {

```